tests/devices/qubit/test_simulate.py

```diff
@@ -189,6 +189,16 @@ def test_result_has_correct_interface(self, op):
         assert qml.math.get_interface(res) == "jax"
         assert qml.math.allclose(res, -1)
 
+    def test_expand_state_keeps_autograd_interface(self):
+        """Test that expand_state doesn't convert autograd to numpy."""
+
+        @qml.qnode(qml.device("default.qubit", wires=2))
+        def circuit(x):
+            qml.RX(x, 0)
+            return qml.probs(wires=[0, 1])
+
+        assert qml.math.get_interface(circuit(1.5)) == "autograd"
```

Comment:
Can we test differentiability too?

Reply:
I just locally raised an Exception in `expand_state_over_wires` and ran `pytest tests/interfaces/default_qubit_2_integration`. Got:
```
1242 failed, 3451 passed
```
I think it's tested 😅